app/components/appcomponents/Footer.js

```
@@ -421,22 +431,34 @@ export default function Footer() {
+                        alt="Facebook Icon"
+                        width={175}
+                        height={42}
+                      />
+                    </a>
+                  </div>
+                  <div className="flex justify-center items-center h-[38px]   ">
+                    <a
+                      href="https://www.facebook.com/osmrtnicacom/"
+                      className="flex items-center text-gray-600  hover:text-blue-500 transition duration-200 "
+                      target="_blank"
+                    >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)